### DISCUSSION TITLE
deleted tables still show up in nocodb GUI

### DISCUSSION THREAD

**Discussion by JS-PradnyaC** [nocodb/nocodb#3304] [Q&A]
I have deleted some tables through backend but they still show up in nocodb project GUI
how to get rid of them ?
please let me know

**Comment by o1lab** [comment]
You would have to [sync the metadata](https://docs.nocodb.com/setup-and-usages/sync-schema).

**Reply by JS-PradnyaC** [reply]
i tried above . i am getting 400 error for the sync api with below response

{"msg":"delete from `nc_views_v2` where `id` = 'vw_4nb8g7ug9auxp5' - Cannot delete or update a parent row: a foreign key constraint fails (`nocodb`.`nc_disabled_models_for_role_v2`, CONSTRAINT `nc_disabled_models_for_role_v2_fk_view_id_foreign` FOREIGN KEY (`fk_view_id`) REFERENCES `nc_views_v2` (`id`))"}
